doc/content/getting-started/migrating-from-v2/configure-ttnctl.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Configure V2 CLI
+weight: 15
+description: Install and configure CLI tools
+---
+
+You will need to use the latest version of `ttnctl`, the CLI for {{% ttnv2 %}}. Follow the [instructions from the TTN documentation][1]. An overview is given below:
+
+Download `ttnctl` [for your operating system][2].
+
+Update to the latest version:
+
+```bash
+$ ttnctl selfupdate
+```
+
+Go to https://account.thethingsnetwork.org and click [ttnctl access code][3].
```

Comment:
This URL gets rendered as text.

Please locally run the docs and look at the pages before asking review

Reply:
Had removed link after it was suggested previously. Markdown rendered as link so I assumed Hugo did the same. Re-added link